source/Ship.cpp

```diff
@@ -3585,6 +3616,96 @@ void Ship::SetTargetSystem(const System *system)
 
 
 
+// Persistent targets for mission NPCs.
+const bool Ship::HasTravelDirective() const
+{
+	return !travelDestinations.empty() || destinationSystem;
+}
+
+
+
+const map<const Planet *, bool> Ship::GetStopovers() const
+{
+	return travelDestinations;
+}
+
+
+
+const System *Ship::GetDestinationSystem() const
+{
+	return destinationSystem;
+}
+
+
+
+// Persistent targets for mission NPCs.
+
+void Ship::SetStopovers(const vector<const Planet *> stopovers, const Planet *destination)
+{
+	// Mark each planet as not visited.
+	if(!stopovers.empty())
+		for(const auto &it : stopovers)
+		{
+			travelDestinations[it] = false;
+			Logger::LogError("Stopover: " + it->Name());
+		}
```

Comment:
Looks like leftover debug code
```suggestion
			travelDestinations[it] = false;
```